### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by christinaholtNOAA**
Updating the README to include current examples of running the graphics package, and a couple of troubleshooting tips and tricks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,45 +3,223 @@
...
+Otherwise, you can retrieve it from HPSS or even the NOAA Big Data Project cloud
+buckets.
+
+An example of pulling a wgrib file for HRRR from BDP:
+
+```
+wget https://noaa-hrrr-bdp-pds.s3.amazonaws.com/hrrr.20220214/conus/hrrr.t00z.wrfprsf06.grib2
+```
+
+Check out list of available NOAA BDP data sets here at [this link](
```
Comment (cshartsough) [inline README.md:64]: "Check out the list..."

File: README.md [hunk 2]
```diff
@@ -3,45 +3,223 @@
...
+         --images ./image_lists/rrfs_subset.yml hourly \
+         -m "My RRFS Retro" \
+         -n ${SLURM_CPUS_ON_NODE:-12} \
+         -o /path/to/output/images \
+         -s 2021052315 \
+         --tiles full,ATL,CA-NV,CentralCA
+
+```
+NOTE: The graphics already run as a workflow step in the RRFS Retros! They may be
+zipped by default, so you can unzip those files to see your images on disk.
```
Comment (cshartsough) [inline README.md:146]: Should we note here that the zipped files get moved over from `pyprd` to `nclprd`?  I remember Eric learned that the hard way. ;-)

**Review by cshartsough — APPROVED** [review]
Nice additional documentation.  Just had one grammar, one suggestion.

I learned some things I didn't know before.  :-)

**Comment by christinaholtNOAA** [comment]
I didn't mention that I briefly tested the examples by copying and modifying them to plot Curtis's sample plots for the new RRFS NA grid.

**Review by Brian-Jamison — APPROVED** [review]
LGTM!  Though we're still currently using NCL for the global FV3, but that will change soon.

**Pull request merged**